api-docs/custom-field-intro.md

```diff
@@ -3,32 +3,58 @@ title: Custom Fields API - Introduction
+With our Custom Fields offering, Trackunit Iris can host all the data that you need. Custom Fields will enable you to capture and organize additional metadata that holds significant contextual value, thereby facilitating more robust data categorization and retrieval processes for you and your end-customers.
 
-Custom fields provide a way to define new fields in the Trackunit data model. Allowing to extend and customize Trackunit Manager. Currently we support **extending the data model of assets, accounts, groups and sites with new fields**. Apart from defining your own custom fields we also provide a number of standard fields for common metadata.
+# Additional standard fields for commom metadata
+Trackunit provides a number of standard fields for common metadata, that can be used by all customers to extend the available data model.
+
+| Field name | Description | Type (UoM) | Related domain |
+| -------- | ------- |-------- | ------- |
+| AC Input Voltage | The input voltage of an alternating current input, could either be to operate directly or to charge a battery. | text (V) | asset |
+| Engine manufaturer | The manufaturer of the engine. | dropdown, single-select | asset |
+| Engine model | The model of the engine. | text | asset |
+| Engine serial number | The serial number of the engine. | text | asset |
+| Engine fuel type | The fuel type the main combustion engine consumes. | dropdown, single-select (Diesel, Petrol, CNG, LPG, HVO100, Hydrogen, Bio-diesel) | asset |
+| Fuel tank capacity (total) | The total fuel capacity, summed over all tanks. | number (L / US Liquid Gallon) | asset |
+| Rated battery voltage | The rated voltage of the traction battery. | text (V) | asset |
+| Rated combustion engine power | Power output of engine in kilo-Watts | text (kW) | asset |
+| Rated electric motor power | Power output of the electric motor in kilo-Watts | text (kW) | asset |
+| Total rated battery capacity | Total rated capacity summed (only if batteries in parallel) over all traction batteries, in ampere-hours. | text (Ah) | asset |
+| Total rated battery power capacity | Total rated energy capacity summed over all traction batteries, in kilo-watt-hours  | text (kWh) | asset |
+| Warranty end date | The end date of the asset's warranty. | date | asset |
+
+## How to set values on standard fields
+
+1. Use the [Custom Fields Values API](/reference/custom-fields-get-values) to get all available fields.
+- Specify the `entityId` for your query, which is an assetId for any fields listed above.
+- Specify which System Of Measurement you would like to use (SI or US customary)
```

Comment:
I see that the docs says System of measurement is required. But that is actually a mistake in the docs. @jbd-trackunit will fix the docs.
It is really only relevant for the fuel tank capacity right now to select gallons vs liter.

Reply:
Great 👍 I have removed the mention from the docs.